File: GLBank/src/glbank/Card.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package glbank;

/**
 *
 * @author Solanid
 */
public class Card {
    private int idCard;
    private long cardNumber;
    private long idAcc;
    private boolean blocked;
    private int pin;

    public Card(int idCard, long cardNumber, long idAcc, boolean blocked, int pin) {
        this.idCard = idCard;
        this.cardNumber = cardNumber;
        this.idAcc = idAcc;
        this.blocked = blocked;
        this.pin = pin;
    }

    public Card(long cardNumber, long idAcc, int pin) {
        this.cardNumber = cardNumber;
        this.idAcc = idAcc;
        this.pin = pin;
        this.idCard = 0;
        this.blocked = false;
    }
    
    public int getIdCard() {
        return idCard;
    }

    public long getCardNumber() {
        return cardNumber;
    }

    public long getIdAcc() {
        return idAcc;
    }

    public boolean isBlocked() {
        return blocked;
    }

    public int getPin() {
        return pin;
    }
    
    
}

File: GLBankATM/GLBankATM/card.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GLBankATM
{
    public class Card
    {
        private int idCard;
        private long cardNumber;
        private long idAcc;
        private bool blocked;

        public Card(int idCard, long cardNumber, long idAcc, bool blocked)
        {
            this.idCard = idCard;
            this.cardNumber = cardNumber;
            this.idAcc = idAcc;
            this.blocked = blocked;
        }

        public int getIdCard()
        {
            return idCard;
        }

        public long getCardNumber()
        {
            return cardNumber;
        }

        public long getIdAcc()
        {
            return idAcc;
        }

        public bool idBlocked()
        {
            return blocked;
        }
    }
}
